src/plugins/Workers/SyncWorker/utils/TransactionOrderer/methods/insertAtPos.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Add a transaction at a specific position
+ * @param {Transaction} transaction
+ * @param {number|null} pos
```

Comment:
Let's make it just optional (undefined)?

Reply:
Totally, fixed!